bin/run-integrationtests

```diff
@@ -12,16 +12,14 @@ function test_auth() {
     cp -av test/datahike_server/resources/config.edn resources/config.edn
     ./bin/kaocha --focus datahike-server.integration-test
 }
-
-function test_dev-mode() {
+function test_dev_mode() {
     cp -av test/datahike_server/resources/config-dev-mode.edn resources/config.edn
-    ./bin/kaocha --focus datahike-server.integration-test-dev-mode
+    ./bin/kaocha --focus datahike-server.setup-test
```

Comment:
Do the names `test/datahike_server/setup_test.clj`, `setup-test/dev-mode-test`, and `test-dev-mode` make sense together, or should one or more of them be changed?

Reply:
At some point this needs some better names and potentially just a refactoring into the integration namespace.

Reply:
I thought a little bit about that and we might add additional tests to the setup test, so things like different backends or env vars. And in the current case we're only testing whether the dev mode works.